android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java

```diff
@@ -266,7 +266,7 @@ private static void repeatNotification(Context context, NotificationDetails noti
                 repeatInterval = 60000 * 60;
                 break;
             case Daily:
-                repeatInterval = 60000 * 60 * 24;
+                repeatInterval = 60000 * 60 * 24 * notificationDetails.dayInterval;
```

Comment:
`notificationDetails.dayInterval` will be null for notifications that had been scheduled before these changes were made so will be need be treated as though it's been set to 1

Reply:
Oh indeed! I'll set the dayInterval to 1 when notificationDetails.dayInterval is null, thanks for spotting this one!